src/components/Button/ButtonAddCard/ButtonAddCard.module.scss

```
@@ -0,0 +1,19 @@
+.buttonAddCard {
+  cursor: pointer;
+  border: none;
```

```suggestion
  border: none;
```
reset.css에 추가하신 내용이라 없어도 될 것 같네요!

---

다시 push할 예정입니다